C-Plus-Plus/math/neon_number.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+	A neon number is a number where :- 
+	the sum of digits of square of the number is equal to the number. 
+	The task is to check and print neon numbers in a range given by the user.
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+//Function that prints the square of a number
+int square(int z)
+{
+    return z * z;
+}
```

Comment:
There is no need to create additional function, C++ supports power in-built function

Reply:
Resolve this @ankan1811